assets/js/highlight.pack.js

```diff
@@ -469,7 +469,13 @@ hljs.registerLanguage('xml', function(s) {
           sL: ['actionscript', 'javascript', 'handlebars', 'xml']
         }
       },
-      {cN: 'meta', v: [{b: /<\?xml/, e: /\?>/, r: 10}, {b: /<\?\w+/, e: /\?>/}]},
+      {
```

Comment:
Doing formatting at the same time? Might be simpler to do this in a follow-up PR...